src/app/components/feed/feeds.service.js

```diff
@@ -13,9 +13,14 @@
 
   FeedsService.$inject = ['$q', '$window'];
 
+  /**
+   * Service containing all feeds
+   * @constructor
+   * @memberof module:janusHangouts
+   */
   function FeedsService($q, $window) {
-    this.mainFeed = null;
-    this.feeds = {};
+    var mainFeed = null;
+    var feeds = {};
```

Comment:
If you change this (which I think is correct), shouldn't you change all the references in the file?